install.sh

```diff
@@ -928,6 +928,24 @@ if [ $targetSystem = raspi ] ; then
     sudo bash $INSTALLERDIR/raspi/access_point/wap.sh
 fi
 
+read -p "Do you want to download languages for offline recognition?[y/n] " lang
```

Comment:
Interaction is not possible, since we do build testing, as well as automated image creation. This needs to be a command line option or some combination.